src/WebJobs.Script.WebHost/Program.cs

```diff
@@ -81,12 +81,26 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args = null)
                 .UseStartup<Startup>();
         }
 
+        internal static void LogVersions(object state)
+        {
+            Console.WriteLine($"HOST_VERSION : {Environment.GetEnvironmentVariable("HOST_VERSION")}");
+            Console.WriteLine($"FUNCTIONS_EXTENSION_VERSION : {Environment.GetEnvironmentVariable("FUNCTIONS_EXTENSION_VERSION")}");
+            Console.WriteLine($"FRAMEWORK : {Environment.GetEnvironmentVariable("FRAMEWORK")}");
+            Console.WriteLine($"FRAMEWORK_VERSION : {Environment.GetEnvironmentVariable("FRAMEWORK_VERSION")}");
+        }
```

Comment:
Do these values change with time? Why do we need to have a timer for this method?

Reply:
These values don't change usually unless the customer explicitly change them in funcapp configuration. Also, if I don't put timer it might get published when the pod/app starts and never again. Although I am totally in favour of increasing the time duration. Please let me know what do you think